### DISCUSSION TITLE
Issue with deploying SO in airgapped environment

### DISCUSSION THREAD

**Discussion by WordlyEagle252** [Security-Onion-Solutions/securityonion#10375] [Unsupported Versions]
I am facing an issue while deploying the SO ISO (version 2.3.240) on vmware in an airgapped setup. I am not getting any option for selecting 'airgap' mode during the installation process. Kindly help me with the complete process to deploy and configure SO for my airgapped network.

**Comment by dougburks** [comment]
When you boot the ISO image, what options are you choosing?

**Reply by WordlyEagle252** [reply]
I'm selecting the first option of 'Install Security  Onion 2.3.240'. I am not shown the airgap option at all. Post installation of CentOS, I'm choosing the 'standalone' mode while performing configuration.

**Reply by dougburks** [reply]
There shouldn't be an airgap option on the ISO boot menu. The airgap option comes later in Setup. It's the 17th screenshot in the screenshot tour at https://docs.securityonion.net/en/2.3/first-time-users.html. That screenshot tour is for IMPORT mode but STANDALONE mode should ask the airgap question at about the same point.

**Reply by WordlyEagle252** [reply]
Thanks for the link. It was really helpful. Can you also help me with the network configuration part. I'm planning to deploy SO on a VM for my airgapped office network. What should be the VMWare settings for the 2 NIC cards. Secondly, any other thing I need to keep in mind while doing the network configuration part?
TIA

**Reply by dougburks** [reply]
If you haven't already, please review the following sections of the documentation:
https://docs.securityonion.net/en/2.3/vmware.html
https://docs.securityonion.net/en/2.3/hardware.html#nic
https://docs.securityonion.net/en/2.3/airgap.html
https://docs.securityonion.net/en/2.3/configuration.html
